domains/transfer-agreements/TransferAgreements.API/API.IntegrationTests/Factories/TransferAgreementsApiWebApplicationFactory.cs

```diff
@@ -48,31 +49,34 @@ protected override IHost CreateHost(IHostBuilder builder)
 
         var serviceScope = host.Services.CreateScope();
         var dbContext = serviceScope.ServiceProvider.GetRequiredService<ApplicationDbContext>();
+
         dbContext.Database.Migrate();
 
         return host;
     }
 
-    public async Task SeedData(Action<ApplicationDbContext> seeder)
+    public async Task SeedData(IEnumerable<TransferAgreement> transferAgreements)
```

Comment:
Will the actor-information be set on the history entries when using the SeedData?

Reply:
No it will not, Audit.net should not be enabled when seeding the database. Seeding the database, is a method that was created, in order to populate the database, without any middleware. Some edgecases which we are testing require the presence of data, that normally never would pass validation checks, if created/inserted through the api. For example Expired transfer agreements.

Audit.Net only hooks onto the ef core dbcontext instance, and only creates audits, when entities are CHANGED, meaning Insert, Update, Delete, Database operations. To circumvent this, we are seeding the database, directly using SQL queries.